examples/api/java/Relations.java

```diff
@@ -0,0 +1,215 @@
+/*********************                                                        */
+/*! \file Relations.java
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Mudathir Mahgoub
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2020 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief Reasoning about relations with CVC4 via Java API.
+ **
+ ** A simple demonstration of reasoning about strings with CVC4 via Jave API.
+ **/
+
+import edu.nyu.acsys.CVC4.*;
+
+/*
```

Comment:
Can you clarify why this is being written here?  Like "This file uses the API to make a sat call equivalent to the following benchmark:".

Reply:
> I believe you need to add the example to the tests makefile here: https://github.com/CVC4/CVC4/blob/master/examples/api/java/CMakeLists.txt
> 
> Then, verify that it passes Travis.

I missed this because I searched for `Strings.java` and not `Strings`. Fixed